main/templates/base/stepper.html

```diff
@@ -6,17 +6,18 @@
         {% for item in stepper.build_stepper %}
             <li>
                 {% if item.is_disabled %}
-                    <div class="stepper-item disabled {{ item.css_classes }}">
+                    <div class="stepper-item disabled {{ item.get_css_classes }}">
                         <span class="stepper-bubble {{ bubble_size.0 }}">{% counter counter value %}</span>
                         <span>{{ item.title }}</span>
                     </div>
                 {% else %}
-                    <a class="stepper-item {{ item.css_classes }}" href="{{ item.get_url }}">
+                    <a class="stepper-item {{ item.get_css_classes }}"
+                       href="{{ item.get_url }}">
                         <span class="stepper-bubble {{ bubble_size.0 }}">{% counter counter value %}</span>
                         <span>{{ item.title }}</span>
                     </a>
                 {% endif %}
-                {% if item.children %}
+                {% if item.children and item.is_available %}
```

Comment:
I like this collapsing implementation a lot more! 

If in testing we decide, for example, that we want to leave the Trajectories items expanded by default, we can easily implement that. Very flexible.